{"success": true, "data": [{"type": "correct", "user": {"id": 980, "name": "xxyysec"}, "challenge_id": 54, "date": "2024-04-26T17:49:10.826935+00:00", "id": 1602, "challenge": {"id": 54, "name": "sanitycheck", "value": 1, "category": "misc"}, "team": {"id": 587, "name": "3x30"}}], "meta": {"count": 1}}
